.eslintrc.base.js

```diff
@@ -20,6 +21,7 @@ module.exports = {
     parser: "babel-eslint",
     ecmaVersion: 2018,
     sourceType: "script",
+    loggerFn: () => {},
```

Comment:
What does this do?

Reply:
Never mind, it's needed to avoid this error:

```
 FAIL  test/lint-verify-fail.test.js
  ● test-lint/ causes errors without eslint-config-prettier › encountered a declaration exception

    SyntaxError: Unexpected token = in JSON at position 0
        at JSON.parse (<anonymous>)

      16 |     { encoding: "utf8", env: getEnv() }
      17 |   );
    > 18 |   const output = JSON.parse(result.stdout);
         |                       ^
      19 | 
      20 |   test("every test-lint/ file must cause an error", () => {
      21 |     expect(output.length).toBe(ruleFiles.length);

      at Suite.parse (test/lint-verify-fail.test.js:18:23)
      at Object.describe (test/lint-verify-fail.test.js:12:1)

```